internal/catalog/internal/types/node.go

```diff
@@ -26,8 +26,14 @@ var (
 
 func RegisterNode(r resource.Registry) {
 	r.Register(resource.Registration{
-		Type:     NodeV1Alpha1Type,
-		Proto:    &pbcatalog.Node{},
+		Type:  NodeV1Alpha1Type,
+		Proto: &pbcatalog.Node{},
+		// TODO: A node should be partition scoped. However its HealthStatus which is
+		// namespace scoped has Node as an owner. We do not support ownership between resources
+		// of differing scope at this time. HealthStatus will probably be split out into two different
+		// types, one for namespace scoped owners and the other for partition scoped owners.
+		// Until that time, Node will remain namespace scoped.
+		Scope:    resource.ScopeNamespace,
```

Comment:
FYI: I needed to set this to ScopePartition to test partition. Maybe we need to do that split soon 🤔